CodeGen/docker_compose/amd/gpu/rocm/README.md

```diff
@@ -0,0 +1,128 @@
+###### Copyright (C) 2024 Advanced Micro Devices, Inc.
+
+# Build and deploy CodeGen Application on AMD GPU (ROCm)
+
+## Build images
+
+### Build the LLM Docker Image
+
+```bash
+### Cloning repo
+git clone https://github.com/opea-project/GenAIComps.git
+cd GenAIComps
+
+### Build Docker image
+docker build -t opea/llm-tgi:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/llms/text-generation/tgi/Dockerfile .
+```
+
+### Build the MegaService Docker Image
+
+```bash
+### Cloning repo
+git clone https://github.com/opea-project/GenAIExamples
+cd GenAIExamples/CodeGen
+
+### Build Docker image
+docker build -t opea/codegen:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f Dockerfile .
+```
+
+### Build the UI Docker Image
+
+```bash
+cd GenAIExamples/CodeGen/ui
+### Build UI Docker image
+docker build -t opea/codegen-ui:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f ./docker/Dockerfile .
+
+### Build React UI Docker image (React UI allows you to use file uploads)
+docker build --no-cache -t opea/codegen-react-ui:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f ./docker/Dockerfile.react .
```

Comment:
Have you tested locally since the test scripts didn't cover react ui test?

Reply:
Yes, react UI was tested manually

Reply:
I've given up on React UI. I decided to use the standard UI in order to pass the tests